web/modules/custom/first_custom_module/src/Controller/BasicPageController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace Drupal\first_custom_module\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+/**
+ * Class for showing the pages.
+ */
+class BasicPageController extends ControllerBase
+{
+
+    /**
+     * Display the markup.
+     *
+     * @return array
+     *   Return markup array.
+     */
+    public function Page1()
+    {
+        $user = \Drupal::currentUser()->getAccountName();
```

Comment:
This kind of service calling is not acceptable in classes.
Please use dependency injection here.